optima/parameters.py

```diff
@@ -373,8 +372,8 @@ def makepars(data, label=None, verbose=2):
             pars['initprev'] = data2prev(data=data, keys=keys, **rawpar) # Pull out first available HIV prevalence point
         elif partype=='popsize': # Population size only
             pars['popsize'] = data2popsize(data=data, keys=keys, **rawpar)
-        elif partype=='timepar': # It's a normal time parameter, e.g. hivtest
-            pars[parname] = data2timepar(data=data, keys=keys, **rawpar)
+        elif partype=='timepar':            
+            pars[parname] = data2timepar(data=data, keys=keys, **rawpar) # Otherwise it's a regular time par, made from data
```

Comment:
ha. you change a thing, you change it back, you change it back again, you forget how it was to begin with, you can't go on, you won't go on, you'll go on.